pkgs/by-name/ni/nitrokey-udev-rules/package.nix

```diff
@@ -6,15 +6,22 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "nitrokey-udev-rules";
-  version = "1.0.0";
+  version = "1.1.0";
 
   src = fetchFromGitHub {
     owner = "Nitrokey";
     repo = "nitrokey-udev-rules";
     rev = "v${finalAttrs.version}";
-    hash = "sha256-uq1+YQg+oe5UFphpy1AdxEYaPFyRle6ffYOPoU6Li28=";
+    hash = "sha256-LKpd6O9suAc2+FFgpuyTClEgL/JiZiokH3DV8P3C7Aw=";
   };
 
+  # The repository already contains the generated rules.  Building would only
+  # re-generate them.
+  dontBuild = true;
```

Comment:
If you want to use the prebuilt artifacts then please set  `meta.sourceProvenance` to reflect this

Reply:
personally i think it is better to regenerate it

Reply:
i did a regenerate with this overlay, feels free to copy it or take inspiration :)
https://github.com/a4blue/own-nix-os-config/blob/87f993831c3aee1aff0971d1b80d898f45821851/overlays/previous.nix